src/5e-SRD-Levels.json

```diff
@@ -8158,5 +8154,974 @@
     },
     "url": "/api/subclasses/lore/levels/14",
     "index": 247
+  },
+  {
+    "level": 1,
+    "feature_choices": [],
+    "features": [
+      {
+        "name": "Bonus Proficiency",
+        "url": "/api/features/bonus-proficiency"
+      },
+      {
+        "name": "Disciple of Life",
+        "url": "/api/features/disciple-of-life"
+      }
+    ],
+    "spellcasting": {},
+    "subclass_specific": {},
+    "class": {
+      "name": "Cleric",
+      "url": "/api/classes/cleric"
+    },
+    "subclass": {
+      "name": "Life",
+      "url": "/api/subclasses/life"
+    },
+    "url": "/api/subclasses/life/levels/1",
+    "index": 248
```

Comment:
Not for this PR, but we might as well swap out the index for `${class.name || subclass.name}-${level}`

Reply:
Agreed. I kept thinking this while I was working on this.

Reply:
Mind cutting another issue! :D